src/chrome/content/rules/7-Eleven.com.xml

```diff
@@ -1,84 +1,22 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://apply.7-eleven.com/ => https://apply.7-eleven.com/: (51, "SSL: no alternative certificate subject name matches target host name 'apply.7-eleven.com'")
-
-	Nonfunctional hosts in *7-eleven.com:
-
-		- blog.franchise ¹
-		- www.realestate ²
-
-	¹ Refused
-	² Dropped
-
-
-	Problematic hosts in *7-eleven.com:
-	
-		- ^ *
-		- careers *
-		- corp *
-		- franchise *
-
-	* Mismatched
-
-
-	Mixed content:
-
-		- Bugs, on;
-
-			- sites from b.collective-media.net *
-			- sites from pubads.g.doubleclick.net
-
-	* Secured by us
-
+	Non-functional hosts
+		Timeout was reached:
+		- 7-eleven.com
+		- realestate.7-eleven.com
+		- www.realestate.7-eleven.com
+
+		SSL peer certificate was not OK:
+		- apply.7-eleven.com
+		- blog.franchise.7-eleven.com
+		- sites.7-eleven.com
 -->
-<ruleset name="7-Eleven.com (partial)" default_off='failed ruleset test'>
-
-	<!--	Direct rewrites:
-				-->
-	<target host="apply.7-eleven.com" />
-	<target host="sites.7-eleven.com" />
+<ruleset name="7-Eleven.com (partial)">
 	<target host="www.7-eleven.com" />
-
-	<!--	Complications:
-				-->
-	<target host="7-eleven.com" />
 	<target host="careers.7-eleven.com" />
 	<target host="corp.7-eleven.com" />
 	<target host="franchise.7-eleven.com" />
 
-		<test url="http://careers.7-eleven.com/careers/Careers.html" />
-		<test url="http://sites.7-eleven.com/franchise/home" />
-
-
-	<!--	Not secured by server:
-					-->
-	<securecookie host="^\.7-eleven\.com$" name="^__qca$" />
-
-
-	<rule from="^http://7-eleven\.com/"
-		to="https://www.7-eleven.com/" />
```

Comment:
Shouldn't we keep this?

Reply:
I am more inclined to remove this rule since `^` gives error code 400 and does not work over http(s).